.github/workflows/push-image.yml

```diff
@@ -0,0 +1,35 @@
+name: push-image
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-check:
+    runs-on: ubuntu-latest
+    name: Go build
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

Comment:
To make it clear for myself, are we introducing these actions to enable multi-platform build?

Reply:
The `qemu` step enables us to do multi platform build in future if we want to. We are not doing cross builds now though. The buildx step however is just the modern equivalent of `docker build` IIRC.

Reply:
I'm just wondering if we should have it (at least the QEMU action), if we're not using it currently, what do you think? I think we can always add it later when we decide to support multi-platform build. I believe we also do not support multi-platform build on other components e.g. API, but we would like to in the future.